hack/build-coreos-manifest.go

```diff
@@ -0,0 +1,58 @@
+// Grafts in the stream json into the configmap stub.
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+)
+
+const (
+	configmapBase = "manifests/coreos-bootimages.json"
+	streamJSON    = "data/data/rhcos-stream.json"
+	dest          = "bin/manifests/coreos-bootimages.json"
+)
+
+func run() error {
+	var cm map[string]interface{}
```

Comment:
Is there some reason why you don't want to have the type here be corev1.ConfigMap? If it were, then we would get type safety later when filling out the `data` and not have to do any casting.

Reply:
In the original repo I wasn't vendoring anything, it was basically just a git repo with a JSON file, so adding all the kube stuff to `vendor/` felt like overkill.

But here, we're already vendoring, so indeed - fixed!